### ISSUE TITLE
Rendered html is inline

### ISSUE THREAD

**Issue by AntonKL** [alchapone/polymer-rails#45]
When I import components in application.html.erb the content of my imports gets imported, not link URLs like it's supposed to work.

__application.html.erb__
```
//= require polymer/polymer
//= require paper-styles/paper-styles-classes
//= require iron-flex-layout/classes/iron-flex-layout
//= require paper-drawer-panel/paper-drawer-panel
//= require paper-header-panel/paper-header-panel
//= require iron-icons/iron-icons
//= require paper-toolbar/paper-toolbar
//= require paper-icon-button/paper-icon-button
//= require paper-menu/paper-menu
//= require paper-material/paper-material
//= require paper-item/paper-item
//= require iron-selector/iron-selector
//= require iron-pages/iron-pages
//= require paper-toast/paper-toast
```

rendered __application.html.__
```
/** @license Copyright (c) 2015 The Polymer Project Authors. All rights reserved. This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as part of the polymer project is also subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt */ :host { display: block; position: absolute; top: 0; left: 0; width: 100%; height: 100%; overflow: hidden; } iron-selector > #drawer { position: absolute; top: 0; left: 0; height: 100%; background-color: white; will-change: transform; box-sizing: border-box; -moz-box-sizing: border-box; @apply(--paper-drawer-panel-drawer-container); } .transition > #drawer { transition: -webkit-transform ease-in-out 0.3s, width ease-in-out 0.3s; transition: transform ease-in-out 0.3s, width ease-in-out 0.3s; } .left-drawer > #drawer { @apply(--paper-drawer-panel-left-drawer-container); } .right-drawer > #drawer { left: auto; right: 0; @apply(--paper-drawer-panel-right-drawer-container); } iron-selector > #main { position: absolute; top: 0; right: 0; bottom: 0; @apply(--paper-drawer-panel-main-container); } .transition > #main { transition: left ease-in-out 0.3s, padding ease-in-out 0.3s; } .right-drawer > #main { left: 0; } .right-drawer.transition > #main { transition: right ease-in-out 0.3s, padding ease-in ------ Cut to save space in this issue -------
```
Everything inline, minified and without the js.

**Comment by alchapone** [comment]
Polymer-rails inlines all css and js into an HTML component and merges the into one single `application.html`. So what exactly is working incorrectly?

**Comment by AntonKL** [comment]
*example app*
![example-app](https://cloud.githubusercontent.com/assets/1410781/7964129/86b360f4-0a16-11e5-996b-a36274c44226.jpg)

*my app*
![myapp](https://cloud.githubusercontent.com/assets/1410781/7964131/88aa5016-0a16-11e5-85be-d9ae581fa283.jpg)

**Comment by alchapone** [comment]
So something in your required elements messes up html. Try to find that line by requiring custom components one by one.

**Comment by AntonKL** [comment]
I find your argument completely valid, and you are true about that there are some components that mess up the html. The problem is that i'm only using standard polymer components from the starter project that works flawless outside rails. Did you try the starter kit etc. when you updated the gem to 1.0? Just curious to see if this is gem related or a rails conflict with something else. Thanks!

**Comment by alchapone** [comment]
I've added a quick fix to v1.0 branch. Try using gem from branch `1.0.x` and let me know if it solves issue for you.

**Comment by AntonKL** [comment]
Still facing issues with rendering. To make sure it's not related to my ruby environment - did you try the Polymer starterkit using 1.0.x?

I'll see if I can test this in a virtual machine also to make sure it's not a conflict.

I also assume that referring to your branch in gemfile should work just fine.
```gem 'polymer-rails', :github => 'alchapone/polymer-rails', :branch => '1.0.x'```

**Comment by alchapone** [comment]
Yes, I've found while trying  to use starterkit. And fixed it in branch `1.0.x`. Most of components from starterkit work fine with that fix. But there might still be some issues with icons components. So yes, specify branch in gemfile, as above, and let me know if there are any issues with some components.
